flutter/shell/platform/tizen/channels/keyboard_channel.cc

```diff
@@ -189,11 +191,25 @@ void KeyboardChannel::SendChannelEvent(const char* key,
     event.AddMember(kTypeKey, kKeyUp, allocator);
   }
   key_event_channel_->Send(
-      event, [this, sequence_id](const uint8_t* reply, size_t reply_size) {
+      event, [this, sequence_id, symbol = std::string(key), is_down](
+                 const uint8_t* reply, size_t reply_size) {
         if (reply != nullptr) {
           std::unique_ptr<rapidjson::Document> decoded =
               JsonMessageCodec::GetInstance().DecodeMessage(reply, reply_size);
           bool handled = (*decoded)[kHandledKey].GetBool();
+          // If System's back key is handled in key-down, it should be
+          // handled so that "popRoute" is not called in key-up.
+          if (symbol == kBackKey) {
+            if (handled && is_down) {
+              this->backkey_handled_ = true;
```

Comment:
`flutter_tizen_view.cc` has the back key handling logic as you might already know. Is that not enough or is it not the right place to handle this case?

Reply:
No, I don't understand the exact issue but I was just wondering why the code was split into two different files. I don't mind if that makes it more readable for you.

Reply:
I moved the handling code for backkey to flutter_tizen_view.cc.